src/app/shell/personal-cabinet/parent/create-child/create-child.component.html

```diff
@@ -12,16 +12,20 @@ <h3 class="wrapper-title">{{(editMode) ? 'РЕДАГУВАТИ' : 'ДОДАТИ'
         [socialGroups]="(socialGroups$ | async)" (deleteForm)="onDeleteForm($event)"></app-child-form>
     </ng-container>
     <div fxLayout='column' fxLayoutAlign='center center'>
-      <button *ngIf="!editMode" class="btn-add" mat-button (click)="addChild()">
+      <button *ngIf="!editMode" class="btn-add" mat-button (click)="addChild()" >
         <mat-icon class="mat-icon-add">add</mat-icon>Додати ще одну дитину
       </button>
-      <div fxLayout="column" fxLayoutAlign="center center">
-        Додаючи дані про дитину я погоджуюсь з <a class="link">Правилами користування </a>
-        та надаю згоду на обробку персональних даних
+
+      <div *ngIf="!isAgreed" fxLayout='column' fxLayoutAlign='center center' class="wrapper-check-box">
+        <mat-checkbox class="step-check-box" color="primary" [formControl]="AgreementFormControl"> Додаючи дані про дитину я погоджуюсь з <a class="link">Правилами користування </a>
+          та надаю згоду на обробку персональних даних
```

Comment:
The text should not be displayed if this is edit mode, and the button should be always enabled